### PR TITLE
Mrd 702 e2e next appointment

### PR DESCRIPTION
**Pull request by bills7281**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: '3.9'
+version: '3.3'
```
Comment (jonnywyatt) [inline docker-compose.yml:1]: not sure why this version is reverted?

File: e2e_tests/integration/recallDeterminate.feature [hunk 2]
```diff
@@ -3,38 +3,38 @@ Feature: Recall (determinate)
   Background:
     Given Maria signs in to the case overview
 
-  Scenario: Not extended / fixed term changed to standard recall
-    Given Maria starts a new recommendation
-    And Maria explains how the person has responded to probation so far
-    And Maria selects the licence conditions that have been breached
-    And Maria selects the alternatives to recall that have been tried
-    And Maria continues from the Stop and Think page
+#   Scenario: Not extended / fixed term changed to standard recall
```
Comment (jonnywyatt) [inline e2e_tests/integration/recallDeterminate.feature:6]: Needs uncommenting

**Review by jonnywyatt — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (bills7281) [reply docker-compose.yml]: My bad! Fixed now!

File: e2e_tests/integration/recallDeterminate.feature (hunk 2 above)
Comment (bills7281) [reply e2e_tests/integration/recallDeterminate.feature]: Done!

**Review by jonnywyatt — APPROVED** [review]

**Pull request merged**